src/main/java/com/yanna/stepanova/service/impl/OrderItemServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.yanna.stepanova.service.impl;
+
+import com.yanna.stepanova.dto.orderitem.OrderItemDto;
+import com.yanna.stepanova.mapper.OrderItemMapper;
+import com.yanna.stepanova.model.CartItem;
+import com.yanna.stepanova.model.Order;
+import com.yanna.stepanova.model.OrderItem;
+import com.yanna.stepanova.repository.order.OrderItemRepository;
+import com.yanna.stepanova.service.OrderItemService;
+import jakarta.persistence.EntityNotFoundException;
+import jakarta.transaction.Transactional;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Transactional
```

Comment:
Apply `@Transactional` only to methods that require transactional behavior, such as those performing create, update, or delete operations
```suggestion
```

Reply:
In the previous task, the mentor advised to use this annotation above a class instead of above method if it's used for each public method. In this case, the same situation.